packages/interpreter/src/components/interpreter.ts

```diff
@@ -15,6 +16,6 @@ export default class Interpreter {
 
   interpret(code: string) {
     const ast = this._parser.parse(code);
-    InterpreterModule.getVisitor(ast.type).visitNode(ast);
+    InterpreterModule.getVisitor(ast.type)?.visitNode(ast);
```

Comment:
Instead of checking for null / undefined everywhere, we can throw an exception from _InterpreterModule.getVisitor_ if the visitor is not present in the module. This will server two purposes:

1. If a developer misses creating a visitor in the module, then he would quickly come to know with the correct exception message.
2. Making branch testing 100% would be much easier as we won't have write negative testcases of missing visitor.